cdap-ui/app/cdap/components/Replicator/Detail/Monitoring/Metrics/index.tsx

```diff
@@ -102,31 +124,65 @@ const MetricsView: React.FC<WithStyles<typeof styles>> = ({ classes }) => {
     );
   }, [timeRange, tables]);
 
+  const parsedLatency = parseLatencySeconds(data.latency);
+
   return (
     <div className={classes.root}>
       <div className={`${classes.grid} ${classes.header}`}>
         <div>Data replicated</div>
-        <div>Throughput events per min</div>
+        <div>Throughput</div>
         <div>Avg latency</div>
-        <div>Pipeline errors</div>
+        <div>Errors</div>
         <div>Total events</div>
         <div>Inserts</div>
         <div>Updates</div>
         <div>Deletes</div>
       </div>
       <div className={`${classes.grid} ${classes.metricContent}`}>
         <div>{data.dataReplicated}</div>
-        <div>{data.eventsPerMin}</div>
-        <div>{data.latency}</div>
+        <div>{truncateNumber(data.eventsPerMin, PRECISION)}</div>
+        <div>
+          <span>{parsedLatency.minutes}</span>
+          <span className={`${classes.latencyLabel} ${classes.marginRight}`}>min</span>
+          <span>{parsedLatency.seconds}</span>
+          <span className={classes.latencyLabel}>sec</span>
+        </div>
         <div className={classes.error}>{data.errors}</div>
-        <div>{data.totalEvents}</div>
-        <div>{data.inserts}</div>
-        <div>{data.updates}</div>
-        <div>{data.deletes}</div>
+        <div>{formatNumberDisplay(data.totalEvents)}</div>
+        <div>{formatNumberDisplay(data.inserts)}</div>
+        <div>{formatNumberDisplay(data.updates)}</div>
+        <div>{formatNumberDisplay(data.deletes)}</div>
+      </div>
+      <div className={`${classes.grid} ${classes.unit}`}>
+        <div />
+        <div>events/min</div>
+        <div />
+        <div />
+        <div />
+        <div />
+        <div />
+        <div />
       </div>
     </div>
   );
 };
 
+function parseLatencySeconds(latency) {
+  const minutes = Math.floor(latency / ONE_MIN_SECONDS);
+  const seconds = latency % ONE_MIN_SECONDS;
+
+  return {
+    minutes,
+    seconds,
+  };
+}
+
+function formatNumberDisplay(num) {
```

Comment:
Seems like there is a similar function added to the replication utility. May be merge them?